app/components/ui-kit/box.js

```diff
@@ -8,10 +8,12 @@ import concat from 'travis/utils/ui-kit/concat';
 const COLORS = {
   WHITE: 'white',
   GREY_LIGHT: 'grey-light',
+  GREY_LIGHTER: 'grey-lighter',
 };
 
 const BG_COLORS = {
   [COLORS.WHITE]: 'white',
+  [COLORS.GREY_LIGHTER]: 'grey-100',
```

Comment:
`grey-light` is `grey-150` in the border colors definition, so I thought a new value like `grey-lighter` might be appropriate here? Or maybe not, perhaps a 100 bg goes well with a 150 border, so maybe `grey light` works for both, though it seems a little weird

Reply:
Sounds good to me too, thank you for the comment @sorta 👍